src/test/isolation2/input/uao/vacuum_while_max_concurrency.source

```diff
@@ -0,0 +1,407 @@
+102: INSERT INTO AO VALUES (1, 1);
+103: INSERT INTO AO VALUES (1, 1);
+104: INSERT INTO AO VALUES (1, 1);
+105: INSERT INTO AO VALUES (1, 1);
+106: INSERT INTO AO VALUES (1, 1);
+107: INSERT INTO AO VALUES (1, 1);
+108: INSERT INTO AO VALUES (1, 1);
+109: INSERT INTO AO VALUES (1, 1);
+110: INSERT INTO AO VALUES (1, 1);
+111: INSERT INTO AO VALUES (1, 1);
+112: INSERT INTO AO VALUES (1, 1);
+113: INSERT INTO AO VALUES (1, 1);
+114: INSERT INTO AO VALUES (1, 1);
+115: INSERT INTO AO VALUES (1, 1);
+116: INSERT INTO AO VALUES (1, 1);
+117: INSERT INTO AO VALUES (1, 1);
+118: INSERT INTO AO VALUES (1, 1);
+119: INSERT INTO AO VALUES (1, 1);
+120: INSERT INTO AO VALUES (1, 1);
+121: INSERT INTO AO VALUES (1, 1);
+122: INSERT INTO AO VALUES (1, 1);
+123: INSERT INTO AO VALUES (1, 1);
+124: INSERT INTO AO VALUES (1, 1);
+125: INSERT INTO AO VALUES (1, 1);
+126: INSERT INTO AO VALUES (1, 1);
+127: INSERT INTO AO VALUES (1, 1);
+1: COMMIT;
+2: COMMIT;
+3: COMMIT;
+4: COMMIT;
+5: COMMIT;
+6: COMMIT;
+7: COMMIT;
+8: COMMIT;
+9: COMMIT;
+10: COMMIT;
+11: COMMIT;
+12: COMMIT;
+13: COMMIT;
+14: COMMIT;
+15: COMMIT;
+16: COMMIT;
+17: COMMIT;
+18: COMMIT;
+19: COMMIT;
+20: COMMIT;
+21: COMMIT;
+22: COMMIT;
+23: COMMIT;
+24: COMMIT;
+25: COMMIT;
+26: COMMIT;
+27: COMMIT;
+28: COMMIT;
+29: COMMIT;
+30: COMMIT;
+31: COMMIT;
+32: COMMIT;
+33: COMMIT;
+34: COMMIT;
+35: COMMIT;
+36: COMMIT;
+37: COMMIT;
+38: COMMIT;
+39: COMMIT;
+40: COMMIT;
+41: COMMIT;
+42: COMMIT;
+43: COMMIT;
+44: COMMIT;
+45: COMMIT;
+46: COMMIT;
+47: COMMIT;
+48: COMMIT;
+49: COMMIT;
+50: COMMIT;
+51: COMMIT;
+52: COMMIT;
+53: COMMIT;
+54: COMMIT;
+55: COMMIT;
+56: COMMIT;
+57: COMMIT;
+58: COMMIT;
+59: COMMIT;
+60: COMMIT;
+61: COMMIT;
+62: COMMIT;
+63: COMMIT;
+64: COMMIT;
+65: COMMIT;
+66: COMMIT;
+67: COMMIT;
+68: COMMIT;
+69: COMMIT;
+70: COMMIT;
+71: COMMIT;
+72: COMMIT;
+73: COMMIT;
+74: COMMIT;
+75: COMMIT;
+76: COMMIT;
+77: COMMIT;
+78: COMMIT;
+79: COMMIT;
+80: COMMIT;
+81: COMMIT;
+82: COMMIT;
+83: COMMIT;
+84: COMMIT;
+85: COMMIT;
+86: COMMIT;
+87: COMMIT;
+88: COMMIT;
+89: COMMIT;
+90: COMMIT;
+91: COMMIT;
+92: COMMIT;
+93: COMMIT;
+94: COMMIT;
+95: COMMIT;
+96: COMMIT;
+97: COMMIT;
+98: COMMIT;
+99: COMMIT;
+100: COMMIT;
+101: COMMIT;
+102: COMMIT;
+103: COMMIT;
+104: COMMIT;
+105: COMMIT;
+106: COMMIT;
+107: COMMIT;
+108: COMMIT;
+109: COMMIT;
+110: COMMIT;
+111: COMMIT;
+112: COMMIT;
+113: COMMIT;
+114: COMMIT;
+115: COMMIT;
+116: COMMIT;
+117: COMMIT;
+118: COMMIT;
+119: COMMIT;
+120: COMMIT;
+121: COMMIT;
+122: COMMIT;
+123: COMMIT;
+124: COMMIT;
+125: COMMIT;
+126: COMMIT;
+127: COMMIT;
+
+1: BEGIN;
+1: INSERT INTO ao SELECT 2, 1;
+2: VACUUM ao;
+3: BEGIN;
+3: INSERT INTO ao SELECT 2, 1;
+4: BEGIN;
+4: INSERT INTO ao SELECT 2, 1;
+5: BEGIN;
+5: INSERT INTO ao SELECT 2, 1;
+1: COMMIT;
+3: COMMIT;
+4: COMMIT;
+5: COMMIT;
+
+SELECT * FROM ao;
+SELECT * FROM gp_ao_or_aocs_seg('ao') ORDER BY segno;
+
+ALTER RESOURCE GROUP admin_group SET CONCURRENCY 20;
```

Comment:
The default value for admin_group is 10, not 20. I suggest to put this query before the first ALTER to be sure that the second ALTER reset the limit correctly
```
SELECT c.value
FROM pg_authid a
  JOIN pg_resgroupcapability c ON c.resgroupid = a.rolresgroup
  JOIN pg_resourcetype t ON t.restypid = c.reslimittype
WHERE a.rolname = current_user
  AND t.resname = 'active_statements';
```

Reply:
I just copied from https://github.com/arenadata/gpdb/blob/6b9f47498afda1b75e307a60a965b40ff594ff57/src/test/isolation2/input/uao/max_concurrency.source#L392

Reply:
That is, the previous test has already overwritten the default value.

Reply:
Ok, let is reset to 20, but the SELECT is still needed to be sure that some other test doesn't change CONCURRENCY to another value

Reply:
If I hardcode the check for 20 in the test, it will fail on a single run.

Reply:
https://github.com/arenadata/gpdb/blob/6b9f47498afda1b75e307a60a965b40ff594ff57/gpdb-doc/markdown/ref_guide/sql_commands/CREATE_RESOURCE_GROUP.html.md?plain=1#L48

Reply:
I removed the resource group change.